# patch gatling.c to use capability headers from the kernel instead of libcap
--- gatling.c.orig	2021-04-04 21:51:17.641001871 +0000
+++ gatling.c	2021-04-04 21:51:49.798095616 +0000
@@ -62,7 +62,7 @@
 
 #ifdef __linux__
 #include <sys/auxv.h>
-#include <sys/capability.h>
+#include <linux/capability.h>
 #endif
 
 char serverroot[1024];
